### PR TITLE
blog post: The new portusctl

### PR DESCRIPTION
**Pull request by mssola**
This new blog post describes why we re-implemented portusctl, and what's
the future for it.

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: _site/blog/index.html [hunk 1]
```diff
@@ -147,24 +165,6 @@ <h2 class="post-title">Portus 2.1 has been released!</h2>
       </div>
     </div>
     
-    <div class="post-preview col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
```
Comment (vitoravelino) [inline _site/blog/index.html]: Was this removed because now it goes on the second page? I dunno know how jekyll works. :)

Comment (mssola) [reply _site/blog/index.html]: No idea. This is done automatically by Jekyll...

File: _posts/2017-11-23-portusctl.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+   [#1299](https://github.com/SUSE/Portus/pull/1299) !). We quickly embraced
+   this idea and started to move things to this new REST API. As of now, this
+   API is not by any means in a stable phase, but it's mature enough so we can
+   work with it more seriously.
+
+While working on [SUSE CaaS
+Platform](https://www.suse.com/products/caas-platform/), we started to
+appreciate more and more one of the handier tools of Kubernetes:
+[kubectl](https://kubernetes.io/docs/user-guide/kubectl/). This tool is, in
+short, a client of the RESP API of Kubernetes. It has a very nice user
```
Comment (monstermunchkin) [inline _posts/2017-11-23-portusctl.md]: REST

File: _posts/2017-11-23-portusctl.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+got from this proof of concept was that `portusctl` was a good candidate, but
+adding this functionality would increase its maintenance burden. Having both the
+old portusctl and this new big feature was not the best scenario, and following
+the containerization trend, we decided that some of the old portusctl had to be
+removed.
+
+## The new portusctl
+
+The [new portusctl](https://github.com/openSUSE/portusctl) is a tool that
+interacts with the REST API in a very similar way than Kubernetes' kubectl:
```
Comment (monstermunchkin) [inline _posts/2017-11-23-portusctl.md]: "[...] REST API very similar to Kubernetes' kubectl:"

**Comment by mssola** [comment]
@monstermunchkin done!

**Pull request merged**